Example/ViewController.swift

```diff
@@ -240,6 +240,9 @@ class ViewController: UIViewController {
     func navigationViewController(navigationService: NavigationService) -> NavigationViewController {
         let route = navigationService.route
         let options = NavigationOptions( navigationService: navigationService)
+        let topBanner = InstructionsCardCollection()
+        topBanner.cardCollectionDelegate = self
+        options.topBanner = topBanner
```

Comment:
I'd rather this not be the _default_ example. Why not put this in as one of the options in the action sheet when you start navigation?

Reply:
Yes, it's best suited as an alternative, considering all factors.